src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
Yes, something like that. Probably "fluid.textfieldSliderVariety" or so would be more intelligible. The default, written in the framework, would read
fluid.makeGradeLinkage("fluid.textfieldSliderVariety", ["fluid.textfieldSlider"], "fluid.textfieldSliderNative");

Then a user who wants to override this can issue

fluid.makeGradeLinkage("fluid.textfieldSliderVariety", ["fluid.textfieldSlider"], "fluid.textfieldSliderJQuery");

in their own file, before they construct any widgets.
The framework can also support variants of this which are just scoped to particular areas of the component tree, but the user would have to currently "roll their own distribution" for this since the signature of fluid.makeGradeLinkage doesn't support this yet.